OtripleS.Web.Api/Controllers/SemesterCourseController.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using OtripleS.Web.Api.Models.SemesterCourses;
+using OtripleS.Web.Api.Models.SemesterCourses.Exceptions;
+using OtripleS.Web.Api.Services.SemesterCourses;
+using RESTFulSense.Controllers;
+
+namespace OtripleS.Web.Api.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class SemesterCourseController : RESTFulController
+    {
+        private readonly ISemesterCourseService semesterCourseService;
+        public SemesterCourseController(ISemesterCourseService semesterCourseService)=>
+            this.semesterCourseService = semesterCourseService;
+
+        [HttpGet]
+        public ActionResult<IQueryable<SemesterCourse>> GetAllSemesterCourse()
+        {
+            try
+            {
+                IQueryable storageSemesterCourse = this.semesterCourseService.RetrieveAllSemesterCourses();
+                return Ok(storageSemesterCourse);
```

Comment:
There should be a new line between **IQueryable storageSemesterCourse** and **return Ok(storageSemesterCourse)**